dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/client/jaxrs/engines/ManualClosingApacheHttpClient43Engine.java

```diff
@@ -763,14 +779,14 @@ public void close()
    // https://github.com/resteasy/Resteasy/pull/2778
    private static File getTempDir() {
        if (System.getSecurityManager() == null) {
-          final Optional<String> value = ConfigurationFactory.getInstance().getConfiguration().getOptionalValue("java.io.tmpdir", String.class);
+          final Optional<String> value = ResteasyConfigProvider.getConfig().getOptionalValue("java.io.tmpdir", String.class);
```

Comment:
I don't like that this was changed to `ResteasyConfigProvider.getConfig()`. `ResteasyConfigProvider` is deprecated and the RestEasy community gave me this snippet to use.

Reply:
eventually, we'll need to migrate to the new ConfigurationFactory SPIs, but right now those don't work with Open Liberty due to the differences in our overlays and integration with our config.